test/parallel/test-fs-mkdir.js

```diff
@@ -187,6 +188,8 @@ function nextdir() {
     assert.strictEqual(err.code, 'ENOTDIR');
     assert.strictEqual(err.syscall, 'mkdir');
     assert.strictEqual(fs.existsSync(pathname), false);
+    // See: https://github.com/nodejs/node/issues/28015
+    assert(err.path);
```

Comment:
This is failing on Windows in the CI, but we should be able to assert the contents (either fully or partially) of `err.path`.